### PR TITLE
Feature/issue11 improve readme with nozzle config

### PR DESCRIPTION
**Pull request by gp510**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,76 @@ uaac -t member add doppler.firehose splunk-nozzle
...
+Job index to tag nozzle's own log events with.
+
+JOB_HOST - 
+Job host to tag nozzle's own log events with.
+
+ADD_APP_INFO - 
+Query API to fetch app details.
+
+API_ENDPOINT - 
+API endpoint address.
```
Comment (cholick) [inline README.md]: Could clarify by specifying "Cloud Foundry API endpoint" - was pinged by someone about confusion which API was in play.

File: README.md [hunk 2]
```diff
@@ -45,6 +45,76 @@ uaac -t member add doppler.firehose splunk-nozzle
...
+
+API_USER - 
+Cloud Foundry admin user name.
```
Comment (cholick) [inline README.md]: `admin` isn't the right word choice. See the setup section describing the proper scopes required.

File: README.md [hunk 3]
```diff
@@ -45,6 +45,76 @@ uaac -t member add doppler.firehose splunk-nozzle
...
+
+API_PASSWORD - 
+Cloud Foundry admin password.
```
Comment (cholick) [inline README.md]: `admin` isn't the right word choice. See the setup section describing the proper scopes required.

File: README.md [hunk 4]
```diff
@@ -45,6 +45,76 @@ uaac -t member add doppler.firehose splunk-nozzle
...
+
+EVENTS - 
+Comma separated list of events to include.
+possible values: ValueMetric,CounterEvent,Error,LogMessage,HttpStartStop,ContainerMetric
+
+EXTRA_FIELDS - 
+Extra fields you want to annotate your events with. 
+
+FIREHOSE_KEEP_ALIVE - 
+Keep Alive duration for the firehose consumer. Default is 25 seconds.
```
Comment (cholick) [inline README.md]: I have reservations about encoding a default value into a README. If the default changes, I doubt the README will get updated.

File: README.md [hunk 5]
```diff
@@ -45,6 +45,76 @@ uaac -t member add doppler.firehose splunk-nozzle
...
+
+SPLUNK_INDEX - 
+The Splunk index events will be sent to. 
+Warning: Setting an invalid index will cause events to be lost.
+
+FLUSH_INTERVAL - 
+Set the interval for flushing to the heavy forwarder. Default is 5 seconds.
+
+CONSUMER_QUEUE_SIZE - 
+Set the internal consumer queue buffer size. Default is 10000.
```
Comment (cholick) [inline README.md]: I have reservations about encoding a default value into a README. If the default changes, I doubt the README will get updated.

File: README.md [hunk 6]
```diff
@@ -45,6 +45,76 @@ uaac -t member add doppler.firehose splunk-nozzle
...
+Warning: Setting an invalid index will cause events to be lost.
+
+FLUSH_INTERVAL - 
+Set the interval for flushing to the heavy forwarder. Default is 5 seconds.
+
+CONSUMER_QUEUE_SIZE - 
+Set the internal consumer queue buffer size. Default is 10000. 
+
+HEC_BATCH_SIZE - 
+Set the batch size for the events to push to HEC (Splunk HTTP Event Collector). Default is 1000.
```
Comment (cholick) [inline README.md]: I have reservations about encoding a default value into a README. If the default changes, I doubt the README will get updated.

File: README.md (hunk 1 above)
Comment (chenziliang) [inline README.md]: +1

**Comment by gp510** [comment]
@cholick - Thanks Matt. I have updated the branch based on your feedback.

**Pull request merged**